src/Endatix.Api/Endpoints/Auth/Login.cs

```diff
@@ -44,7 +44,7 @@ public override async Task HandleAsync(LoginRequest request, CancellationToken c
         }
         else
         {
-            var successfulResponse = new LoginResponse(request.Email, result.Value.Token, string.Empty);
+            var successfulResponse = new LoginResponse(request.Email, result.Value.AccessToken.Token, result.Value.RefreshToken.Token);

```

Comment:
I am getting a bit nervious when there are 4 items chained even if payload is validated. Might be good to discuss this and set some guideline . I read more about it and it seems what we can do is to leverage the Nullable reference types more and do something like:
>  `result.Value!.AccessToken!.Token!` - need to dig deeper. Here is a read - https://wildermuth.com/2023/02/13/nullable-reference-types-in-csharp/

Not a TODO again. Just sharing

Reply:
I'm not sure if we really need to dig into this. All the values are not nullble, not just because the command ensures they are set correctly but also they are defined as not nullable. Technically we can use the ! operator but it doesn't bring anything and makes the code a bit less cleaner (though just a bit).
Generally, such kind of issues are pointed very clear by the IDE and IMO it we can focus on details that could be problematic for the correctness of how the product works or any general issues with the implementation, especially when they lead to complications with the code maintainability and extension.